drivers/ipm/ipm_sedi.c

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright (c) 2020-2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT intel_sedi_ipm
+
+#include <zephyr/kernel.h>
+#include <zephyr/init.h>
```

Comment:
it seems to have several useless header files include, please remove.

Reply:
done